content/en/docs/guides/keepalive.md

```diff
@@ -0,0 +1,101 @@
+{{% /alert %}}
+
+### How configuring keepalive affects a call
+
+For unary RPC with quick replies, keepalive is less likely to be triggered. Keepalive would primarily be triggered when there is a long-lived RPC, which will fail if keepalive check failed and the connection was closed.
+
+For steaming RPC, if the connection is closed, any in-progress RPCs will fail. If a call is streaming data, the stream will also be closed and any data that has not yet been sent will be lost.
+
+### How to enable and configure keepalive
+
+Based on language, keepalive can be configured by:
+* `ServerOption` and client `DialOption` in Go.
+* Channel and server builder in Java.
+* Channel and server options in C++ and Pyhton.
+
+{{% alert title="Warning" color="warning" %}}
+  To avoidling DDoS, it's important to take caution while set the keepalive cofigurations. Thus, it is recommended to disables keepalive for HTTP/2 connections with no outstanding streams and for clients to avoid configuring their keepalive much below one minute.
+{{% /alert %}}
+
+### Common situations where keepalives can be useful
+
+gRPC HTTP/2 keepalives can be useful in a variety of situations, including but not limited to:
+
+* When streaming data over a long-lived connection which might be considered as idle by proxy or load balancers.
+* When connections are expected to be short-lived.
+* When network is less reliable (For exmaple, mobile applications).
+
+### Keepalive configuration specification
+
+| Options | Availability | Description | Client Default | Server Default |
+|---|---|---|---|---|
+| KEEPALIVE_TIME | Client and Server | The interval in milliseconds between PING frames. | INT_MAX (Disabled) | 27200000 (2 hours) |
+| KEEPALIVE_TIMEOUT | Client and Server | The timeout in milliseconds for a PING frame to be acknowledged. If sender does not receive an acknowledgment within this time, it will close the connection. | 20000 (20 seconds) | 20000 (20 seconds) |
+| KEEPALIVE_WITHOUT_CALLS | Client | Is it permissible to send keepalive pings from the client without any outstanding streams. | 0 (false) | N/A |
+| PERMIT_KEEPALIVE_WITHOUT_CALLS | Server | Is it permissible to send keepalive pings from the client without any outstanding streams. | N/A | 0 (false) |
+| PERMIT_KEEPALIVE_TIME | Server | Minimum allowed time between a server receiving successive ping frames without sending any data/header frame. | N/A | 300000 (5 minutes) |
+| MAX_CONNECTION_IDLE | Server | Maximum time that a channel may have no outstanding rpcs, after which the server will close the connection. | N/A | INT_MAX (Infinite) |
+| MAX_CONNECTION_AGE | Server | Maximum time that a channel may exist. | N/A | INT_MAX (Infinite) |
+| MAX_CONNECTION_AGE_GRACE | Server | Grace period after the channel reaches its max age. | N/A | INT_MAX (Infinite) |
```

Comment:
```suggestion
| `KEEPALIVE_TIME` | Client and Server | The interval in milliseconds between PING frames. | INT_MAX (Disabled) | 27200000 (2 hours) |
| `KEEPALIVE_TIMEOUT` | Client and Server | The timeout in milliseconds for a PING frame to be acknowledged. If sender does not receive an acknowledgment within this time, it will close the connection. | 20000 (20 seconds) | 20000 (20 seconds) |
| `KEEPALIVE_WITHOUT_CALLS` | Client | Is it permissible to send keepalive pings from the client without any outstanding streams. | 0 (false) | N/A |
| `PERMIT_KEEPALIVE_WITHOUT_CALLS` | Server | Is it permissible to send keepalive pings from the client without any outstanding streams. | N/A | 0 (false) |
| `PERMIT_KEEPALIVE_TIME` | Server | Minimum allowed time between a server receiving successive ping frames without sending any data/header frame. | N/A | 300000 (5 minutes) |
| `MAX_CONNECTION_IDLE` | Server | Maximum time that a channel may have no outstanding rpcs, after which the server will close the connection. | N/A | INT_MAX (Infinite) |
| `MAX_CONNECTION_AGE` | Server | Maximum time that a channel may exist. | N/A | INT_MAX (Infinite) |
| `MAX_CONNECTION_AGE_GRACE` | Server | Grace period after the channel reaches its max age. | N/A | INT_MAX (Infinite) |
```

This fixes the issue with unwanted word break, like this one:
<img width="601" alt="image" src="https://github.com/grpc/grpc.io/assets/672669/c654fc33-32ad-4e48-815e-6b60283fa584">

And really improves the looks:
<img width="1183" alt="image" src="https://github.com/grpc/grpc.io/assets/672669/bfefcfd3-77df-4ccb-a2b5-ee231b057996">